app/uk/gov/hmrc/cdsimportsddsfrontend/domain/DocumentationType.scala

```diff
@@ -16,41 +16,57 @@
 
 package uk.gov.hmrc.cdsimportsddsfrontend.domain
 
+case class WriteOffType(quantity: Option[String])
+
+case class AdditionalDocumentType(
```

Comment:
The name shouldn't include the word Type

Reply:
Agree, but this follows the existing pattern. This can be done as part of a refactor after this ticket.